queries/he.sparql

```diff
@@ -0,0 +1,129 @@
+		wd:Q17155032 # type of software
+		wd:Q1725664  # terminology of any kind
+		wd:Q17537576 # any creative work incl. music, video, text, etc.
+		wd:Q17638537 # routine - fix computer terminology
+		wd:Q1897960   # trotting horse
+		wd:Q19798642 # WD value
+		wd:Q19798644 # wikibase model terms
+		wd:Q19798645 # wikibase datatype
+		wd:Q20202269 # music terms
+		wd:Q208569   # album
+		wd:Q21191270 # tv series episode
+		wd:Q215380   # bands
+		wd:Q2235308  # types of ships
+		wd:Q23786    # eye color
+		wd:Q3305213  # painting
+		wd:Q3331189   # version, edition, or translation
+		wd:Q3491429  # military terms
+		wd:Q355304    # watercourse (this has a lot of false positives that are not relevant in a Swedish lexeme context)
+		wd:Q3744866  # common charge (heraldic)
+		wd:Q386724    # work
+		wd:Q395      # math
+		wd:Q40056    # software
+		wd:Q4167410  # disambigpage
+		wd:Q4167836  # wikimedia cat
+		wd:Q43229    # organization of any kind
+		wd:Q482994   # album
+		wd:Q494452   # heraldry
+		wd:Q5        # human
+		wd:Q5185279  # poem
+		wd:Q56876983 # symbol (heraldic)
+		wd:Q57814795 # domesticated animal
+		wd:Q58408484  # Wikimedia list of persons by surname
+		wd:Q7187     # gene
+		wd:Q7366     # song
+		wd:Q7397     # software
+		wd:Q8134     # economics
+		wd:Q819989    # lunar crater
+		wd:Q99045339     # written question
+		wd:Q99281788 # star trek location
+        wd:Q29168811 # animated film
+        wd:Q5398426  # tv series
+        wd:Q3744880  # nature figure (heraldry)
+        wd:Q6881511  # enterprise
+        wd:Q3658341  # literary character
+        wd:Q1475691  # mars crater
+        wd:Q1348589  # lunar crater
+        wd:Q20643955 # biblical human figure
+        wd:Q134556   # single (music)
+        wd:Q786820   # automobile manuf.
+        wd:Q11060274 # print
+        wd:Q18218093 # other type of art
+        wd:Q7725634  # literary work
+  }.
+    ?item wdt:P31 ?not  }.
+  # filter on properties:
+  FILTER NOT EXISTS {
+    VALUES ?not2 {
+      wdt:P688 # gene stuff
+      wdt:P195 # collection (of art)
+      wdt:P1482 # stack exchange
+      }
+      ?item ?not2 [] }.
+  # filter on subclass:
+  FILTER NOT EXISTS {
+    VALUES ?not3 {
+      wd:Q783794  # company
+      }
+      ?item wdt:279 ?not3 }.
+  # Aspect of
+  FILTER NOT EXISTS {
+    VALUES ?aspect_of {
+      wd:Q18336  # heraldry
+      wd:Q14659  # coat of arms
+      }
+    ?item wdt:P1269 ?aspect_of }. 
+  # special terminology often match these descriptions
+  FILTER (!regex(?desc, '.*theory.*')) .
+  FILTER (!regex(?desc, '.*mathemat.*')) .
+  FILTER (!regex(?desc, '.*geolog.*')) .
+  FILTER (!regex(?desc, '.*anatomy.*')) .
+  FILTER (!regex(?desc, '.*military.*')) .
+  FILTER (!regex(?desc, '.*taxonomic.*')) .
+  FILTER (!regex(?desc, '.*C++.*')) .
+  FILTER (!regex(?desc, '.*Unix.*')) .
+  FILTER (!regex(?desc, '.* law .*')) .
+  FILTER (!regex(?desc, '.*heraldic.*')) .
```

Comment:
These don't make sense since the description is Hebrew – right?

Reply:
good catch!